curve25519-dalek/src/lemmas/common_lemmas/mul_lemmas.rs

```diff
@@ -187,6 +187,25 @@ pub proof fn lemma_mul_distributive_8_terms(
     }
 }
 
+pub proof fn distribution_over_8_terms_other_way(
```

Comment:
```suggestion
pub proof fn lemma_mul_distributive_8_terms_other_way(
```
though I'd prefer if you just extended the ensures clauses on `lemma_mul_distributive_8_terms`
https://github.com/Beneficial-AI-Foundation/dalek-lite/blob/05af4cc4f196fc145267341209fd1f8b334f4791/curve25519-dalek/src/lemmas/common_lemmas/mul_lemmas.rs#L170-L172